src/index.js

```diff
@@ -71,15 +73,15 @@ function createBabelPluginFactory(customCallback = returnObject) {
 					return;
 				}
 
-				return babel.buildExternalHelpers(externalHelpersWhitelist, 'module');
+				return babel.buildExternalHelpers(null, 'module');
```

Comment:
What's the rationale for dropping `externalHelpersWhitelist` support? Is it never used in the wild, or no longer valid? I'm all for simplicity here, so :+1: from me :)

Reply:
I'm aiming for simplicity here + rollup should be able to tree shake the generated module, so dont think that there is a strong need for this whitelist.

Reply:
makes sense to me, thanks!